person_capture/gui_app.py

```
@@ -2527,6 +2527,32 @@ def _square_rescue_allowed() -> bool:
+            rs: str,
+            profile: str,
+            actual_face_h_frac: float,
+        ) -> None:
+            nonlocal best_portrait_alt
+            if face is None:
+                return
+            if profile not in {"close", "portrait_close", "upper"}:
+                return
+            if rs not in {"2:3", "3:4"}:
+                return
+            cx1, cy1, cx2, cy2 = [float(v) for v in crop]
```

Ruff (0.15.12)

[warning] Unpacked variable `cy1` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)